docs/dev/guidelines/server.rst

```diff
@@ -137,14 +138,143 @@ Additional notes on the controller methods:
     * Bad Request - the request was wrong.
     * Not Found - can't find the requested data or it should be not accessible yet.
 
-12. Dependency injection
+.. _server-guideline-dto-usage:
+
+12. Use DTOs for data transfer
+==============================
+
+Use data transfer objects (DTOs) to send data from the server to the client, i.e. responses of RestControllers and WebSocket messages. A DTO ...
```

Comment:
I'm not sure what the point of this `...` is

Reply:
Originally, the layout was a bit different. I forgot do delete "A DTO ...". I deleted it now.